spec/views/collections/edit.html.erb_spec.rb

```diff
@@ -2,12 +2,17 @@
 
 RSpec.describe 'collections/edit', type: :view do
   before(:each) do
-    @collection = assign(:collection, create(:collection))
+    @collection = assign(:collection, create(:collection, workspaces: [create(:workspace)]))
   end
 
   it 'renders the edit collection form' do
     render
     expect(rendered).to have_css 'input[name="collection[name]"]'
     expect(rendered).to have_css 'input[type="submit"]'
   end
+
+  it 'renders the related manifest list' do
+    render
+    expect(rendered).to have_css 'h4', text: 'Workspaces'
+  end
```

Comment:
Now that I know how to create associated objects, I'll check for a specific workspace name.